operations/enterprises.md

```diff
@@ -644,15 +662,23 @@ Updates details of the resources.
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the assignment. |
```

Comment:
There's bad previous merge - Resource category on line 640 is nested under response for `resources/update` operation.
Move range of lines 640 - 732 above to line 562.

Reply:
Also the DTOs aren't in same order as in API @prazi

Reply:
so we can merge this one and reorder after that, no? It would just complicated this PR